lerna.json

```diff
@@ -1,6 +1,12 @@
 {
-  "packages": [
-    "packages/*"
-  ],
-  "version": "independent"
+  "version": "independent",
+  "packages": ["packages/*"],
+  "command": {
+    "version": {
+      "conventionalCommits": true
+    },
+    "publish": {
+      "ignoreChanges": ["package-lock.json"]
+    }
+  }
```

Comment:
interesting that it doesn't ignore package-lock by default. already LGTMed in another comment, but still LGTM.

Reply:
Oh, I may need to relative path that.

Reply:
Just pushed a fix for this.

Reply:
took this for a spin with a dummy commit to package-lock.json. lerna was cool with it.